src/instructlab/model/list.py

```diff
@@ -32,9 +39,54 @@
 )
 def model_list(model_dirs: List[str], list_checkpoints: bool):
     """Lists models"""
+    print(f"All listed models can be found at: {DEFAULTS.MODELS_DIR}")
+
     # click converts lists into tuples when using multiple
     model_path_dirs = [Path(m) for m in model_dirs]
 
-    data = list_models(model_path_dirs, list_checkpoints)
-    data_as_lists = [list(item) for item in data]  # this is to satisfy mypy
-    print_table(["Model Name", "Last Modified", "Size"], data_as_lists)
+    valid_models = list_models(model_path_dirs, list_checkpoints)
+    print(valid_models)
```

Comment:
How does this differ from printing the table below?

Reply:
accidental push, removed thanks